rapid7_insightvm/plugin.spec.yaml

```diff
@@ -2182,6 +2182,24 @@ actions:
         description: 'Tag search criteria - options documentation: https://help.rapid7.com/insightvm/en-us/api/#section/Responses/SearchCriteria'
         type: object
         required: true
+      size:
+        title: Size
+        description: The number of records to retrieve. If blank or '0' all assets that match the search will be returned
+        type: number
+        required: false
+        example: 100
+      sort_criteria:
```

Comment:
Both sort criteria and sort order could be made a bit more simple by reducing them to a single JSON object input, eg:

```
{
  "risk-score": "asc",
  "host-type": "desc"
}
```
etc